barber-compiler/build.gradle

```diff
@@ -5,6 +5,7 @@ sourceCompatibility = rootProject.ext.sourceCompatibilityVersion
 targetCompatibility = rootProject.ext.targetCompatibilityVersion
 
 dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Oh wait, I see what you've done here. This actually won't work because it's against the license to distribute that jar. We'll have to match on class naming instead.